src/utils/getMTime.js

```diff
@@ -0,0 +1,3 @@
+import { statSync } from 'fs'
+
+export default filename => statSync(filename).mtime.getTime().toString()
```

Comment:
Decided to put this in its own module so that it would be easy to mock in this plugin‘s tests.